coalib/bearlib/aspects/Spelling.py

```diff
@@ -1,72 +1,64 @@
-from coalib.bearlib.aspects import Root
+from coalib.bearlib.aspects import Root, Taste
 
 
 @Root.subaspect
 class Spelling:
-    """
-    This aspect describes spelling of your source code.
+    """How words should be written.
     """
     class docs:
         example = """
-        `Coala`, `AspectsYEAH`
+        'Tihs si surly som incoreclt speling.
+        `Coala` is always written with a lowercase `c`.
         """
-        example_language = 'All'
+        example_language = 'English'
         importance_reason = """
-        They are words we always want to be written the way we want like the
-        name of our app eg: `coala`...
+        Words should always be written as they are supposed to be,
```

Comment:
The `,` at the end of this line is a grammar error. with the next line being a "run-on phase".
Either it should be a semi-colon, or a full stop.